mathjax3-ts/input/tex/enclose/EncloseConfiguration.ts

```diff
@@ -0,0 +1,79 @@
+/*************************************************************
+ *
+ *  Copyright (c) 2018 The MathJax Consortium
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Configuration file for the enclose package.
+ *
+ * @author v.sorge@mathjax.org (Volker Sorge)
+ */
+
+import {Configuration} from '../Configuration.js';
+import TexParser from '../TexParser.js';
+import {TexConstant} from '../TexConstants.js';
+import {CommandMap} from '../SymbolMap.js';
+import {ParseMethod} from '../Types.js';
+import ParseUtil from '../ParseUtil.js';
+
+
+/**
+ * The attributes allowed in \enclose{notation}[attributes]{math}
+ * @type {{[key: string]: number}}
+ */
+export const ENCLOSE_OPTIONS: {[key: string]: number} = {
+  'data-arrowhead': 1,
+  color: 1,
+  mathcolor: 1,
+  background: 1,
+  mathbackground: 1,
+  'data-padding': 1,
+  'data-thickness': 1
+};
+
+
+// Namespace
+export let EncloseMethods: Record<string, ParseMethod> = {};
+
+
+/**
+ * Implements \enclose{notation}[attr]{math}
+ * (create <menclose notation="notation">math</menclose>)
+ * @param {TexParser} parser The current tex parser.
+ * @param {string} name The name of the calling macro.
+ */
+EncloseMethods.Enclose = function(parser: TexParser, name: string) {
+  let notation = parser.GetArgument(name).replace(/,/g, ' ');
+  const attr = parser.GetBrackets(name, '');
+  const math = parser.ParseArg(name);
+  const def = ParseUtil.keyvalOptions(attr, ENCLOSE_OPTIONS);
+  def.notation = notation;
+  if (def.arrow) {
+    def.notation += ' updiagonalarrow';
+    delete def.arrow;
+  }
```

Comment:
I think this is the same as my comments about `updiagonalarrow` as above.  It should be `northeastarrow`, and since `arrow` is not an allowed option (it's not in `ENCLOSE_OPTIONS`), I don't think this `if-then` will ever be triggered, in any case.  I think you can safely remove it.

We should probably keep a list of these changes from v2 so that they can be documented later.  I've started one, so if you have any others that you know of, let me know.

Reply:
Removed.